plugins/psv-extractor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@flatfile/plugin-psv-extractor",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
Can we do 1.0.0 for everything?

Reply:
i want to do development stage first

Reply:
https://flatfile.com/docs/developer-tools/versioning